function_calling/src/tools.py

```diff
@@ -152,40 +152,95 @@ class ReplSchema(BaseModel):
 ## SQL tool
 # tool schema
 class QueryDBSchema(BaseModel):
-    "A database querying tool. Use this to generate sql querys and retrieve the results from a database. Input should be a natural language question to the db."
+    "A query generation tool. Use this to generate sql querys and retrieve the results from a database. Do not pass sql querys directly Input must be a natural language question or instruction."
```

Comment:
Plural of query is queries.
Period after directlu.

Reply:
fixed